### ISSUE TITLE
XMLString returning plist

### ISSUE THREAD

**Issue by azplanlos** [nicklockwood/XMLDictionary#46]
XMLDictionary's [NSDictionary XMLString] method returns a plist XML on OS X 10.10.5 but returns a correct XML on OS X 10.7.5.

**Comment by sweetppro** [comment]
did you ever figure out the cause of this?
I get the same thing on OSX 10.11.1

**Comment by azplanlos** [comment]
Yes, Apple implemented a similar method with the same name -(NSData*)XMLData. I was able to use -(NSData*)innerXML instead in my case where I don't need the root object. But this might be different in your project. The only way to get rid of this would be to rename the method in the XMLDictionary category which will break old projects. That's why I didn't file a pull request yet.

**Comment by sweetppro** [comment]
ah -  you rock! thanks!
Ive been trying to figure this out all day.
I took your advice and renamed:
- (NSString *)XMLString;
To
- (NSString *)XMLDictionaryString;

then it worked right away.
:)

this fix should really be added to Master, so it doesn't trip other ppl up.
thanks again,
such a simple fix

**Comment by RobertLeeL** [comment]
> :)

awasome ! you are right, that is slove my problem

**Comment by Workshed** [comment]
I've just hit this exact issue on a legacy application that I occasionally work on, thank you for pointing out this issue!
